integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/mail/EduMailSender.java

```diff
@@ -0,0 +1,65 @@
+package no.difi.meldingsutveksling.mail;
+
+import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
+import no.difi.meldingsutveksling.domain.MeldingsUtvekslingRuntimeException;
+
+import javax.activation.DataHandler;
+import javax.mail.*;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
+import javax.mail.util.ByteArrayDataSource;
+import java.util.Properties;
+
+public class EduMailSender {
+
+    private IntegrasjonspunktProperties properties;
+
+    public EduMailSender(IntegrasjonspunktProperties properties) {
+        this.properties = properties;
+    }
+
+    public void send(byte[] messageAsBytes, String title) {
+        Properties props = new Properties();
+        props.put("mail.smtp.host", properties.getMail().getSmtpHost());
+        props.put("mail.smtp.port", properties.getMail().getSmtpPort());
+        props.put("mail.smtp.starttls.enable", "true");
+        props.put("mail.smtp.auth", properties.getMail().getEnableAuth());
+
+        Session session = Session.getDefaultInstance(props,
+                new javax.mail.Authenticator() {
+                    protected PasswordAuthentication getPasswordAuthentication() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1161)